docker_db_config_template/2025.1.3/EXAConf

```diff
@@ -0,0 +1,213 @@
+[Global]
+    Revision = 16
+    Checksum = b859548089afd15a9f0db789233a66c6
+    ClusterName = cl4
+    Platform = Docker
+    LicenseFile = /exa/etc/license.exasol_license
+    CoredPort = 3250
+    SSHPort = 20002
+    IssueTracker = Salesforce
+    XMLRPCPort = 443
+    RESTAPIPort = 8000
+    RESTAPIEnabled = True
+    RESTAPIPathPrefix = /api/v1
+    RESTAPIAuthPubKey = 
+    WebUIBackendPort = 4444
+    WebUIDomain = exacluster.local
+    EnableLoadBalancer = False
+    AuthenticationToken = OUlGWmxvcDRQb2h5WHpDUjpXT3NuNEdwQWZiN3pUczJwRENad3pqZUxzZDJMb0hOejlEeXBMdHVYSUZIbklMZVFMWHR6T1kxMEtOVXJPRk9i
+    # List of networks for this cluster: 'private' is mandatory, 'public' is optional.
+    Networks = private
+    # Comma-separated list of nameservers for this cluster.
+    NameServers = 8.8.8.8
```

Comment:
Did you merge the EXAConf correctly?
Looks like the Jinja template variables were removed, like here.